scripts/build-index.js

```diff
@@ -10,30 +10,27 @@ function parsePagename(pagefile) {
   return pagefile.split(/\//)[2].replace(/\.md$/, '');
 }
 
-function buildShortPagesIndex(files) {
+function parseLanguage(pagefile) {
+  var language = pagefile.split(/\//)[0].replace(/^pages/, '');
+  return language == '' ? 'en' : 'ta';
```

Comment:
This will not work for future languages. Please just return 'en' if the language variable is empty. Else return the value.